sample/lit-todo/index.html

```diff
@@ -1,76 +1,160 @@
+            // https://github.com/rocicorp/replicache#server-side for more
+            // information.
+            diffServerURL: 'https://serve.replicache.dev/pull',
+
+            // Auth token for the diff server, if any.
+            diffServerAuth: '1',
+
+            // URL of your service's Replicache batch endpoint. Replicache
+            // will send batches of mutations here for application when the
+            // network is available.
+            batchURL: 'https://replicache-sample-todo.now.sh/serve/replicache-batch',
+
+            // Auth token for your client view and batch endpoints, if any.
+            dataLayerAuth: '2',
+        });
+
+        rep.onSync = syncing => {
+            const icon = document.querySelector('.sync-icon');
+            icon.textContent = 'sync';
+            icon.classList.toggle('offline', false);
+            if (syncing) {
+                icon.classList.toggle('syncing', true);
+                window.setTimeout(() => icon.classList.toggle('syncing', false), 500);
+            }
+            if (!rep.online) {
+                icon.classList.toggle('offline', true);
+                icon.textContent = 'sync_problem';
+            }
+        };
+
+        const key = id => `/todo/${id}`;
+
+        const updateTodo = rep.register('updateTodo', async (tx, changes) => {
+            const todo = await tx.get(key(changes.id));
+            Object.assign(todo, changes);
+            await tx.put(key(todo.id), todo);
+        });
+
+        const deleteTodo = rep.register('deleteTodo', async (tx, { id }) => {
+            await tx.del(key(id));
+        });
+
+        const createTodo = rep.register('createTodo', async (tx, todo) => {
+            await tx.put(key(todo.id), todo);
+        });
+
+        // TODO: Remove these by auto-syncing after mutate.
+        const handleCheckbox = async (id, e) => {
+            await updateTodo({ id, complete: e.srcElement.selected });
+            rep.sync();
+        };
+
+        const handleDelete = async (id, e) => {
+            cancelEvent(e);
+            await deleteTodo({ id });
+            rep.sync();
+        };
+
+        const cancelEvent = e => {
+            e.stopPropagation();
+        }
+
+        const handleCreate = async () => {
+            const first = (await rep.query(tx => toArray(tx.scan({ prefix: '/todo/', limit: 1 }))))[0];
+            await createTodo({
+                id: Math.round(Math.random() * Math.pow(2, 30)),
+                listID: 2,
+                text: 'Untitled',
+                order: first ? (first.order / 2) : 0.5,
+                complete: false,
+            });
+            document.querySelector('mwc-check-list-item .text').focus();
+            rep.sync();
+        };
+
+        const handleFocus = e => {
+            const selection = window.getSelection();
+            const range = document.createRange();
+            range.selectNodeContents(e.srcElement);
```

Comment:
srcElement is IE-ism. We use target today

Reply:
Fixed in two places.